nomenclature/cli.py

```diff
@@ -93,3 +96,31 @@ def cli_valid_project(
     """
     assert_valid_yaml(path)
     assert_valid_structure(path, definitions, mappings, required_data, dimensions)
+
+
+@cli.command("run-region-processing")
+@click.argument("input_data_file", type=click.Path(exists=True, path_type=Path))
```

Comment:
Suggest to use shorter arg names, similar to pyam...
```suggestion
@click.argument("data", type=click.Path(exists=True, path_type=Path))
```